modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/Conditions.tsx

```diff
@@ -201,7 +201,7 @@ export function Conditions({
 										objectValidationRuleSettings: [
 											{
 												name:
-													'objectFieldExternalReferenceCode',
+													'outputObjectFieldExternalReferenceCode',
```

Comment:
@naraandrade Can we make it a constant?
It's used in three differrent places.

Reply:
See my response above.